src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -27,6 +27,19 @@ class MethodParam {
     }
 }
 
+class ActionParam {
+    public groupOpActionName: string;
+    public groupActionName: string;
+    public action: Parameter;
+    public actionName: string;
+    public constructor(groupOpActionName, groupActionName, actionName, action) {
```

Comment:
>groupOpActionName [](start = 23, length = 17)

you can add public so that typescript will help to make it to a class member. like public constructor (public groupOpActionName, ...)

Reply:
ohh okay